shotgun_api3/shotgun.py

```diff
@@ -3327,17 +3327,39 @@ def _call_rpc(self, method, params, include_auth_params=True, first=False):
         if self.config.localized is True:
             req_headers["locale"] = "auto"
 
-        http_status, resp_headers, body = self._make_call("POST", self.config.api_path,
-                                                          encoded_payload, req_headers)
-        LOG.debug("Completed rpc call to %s" % (method))
-        try:
-            self._parse_http_status(http_status)
-        except ProtocolError as e:
-            e.headers = resp_headers
-            # 403 is returned with custom error page when api access is blocked
-            if e.errcode == 403:
-                e.errmsg += ": %s" % body
-            raise
+        attempt = 1
+        max_attempts = 4 # Three retries on failure
+        backoff = 0.75 # Seconds to wait before retry, times the attempt number
```

Comment:
Should we make this configurable?

Reply:
@jfboismenu -- yes, if it resolves the reported issue I will make it configurable before it's released.